pkgs/applications/version-management/gitlab/gitlab-shell/remove-hardcoded-locations.patch

```diff
@@ -25,7 +25,7 @@ index 435cb29..078c1df 100644
  
  func NewFromDir(dir string) (*Config, error) {
 -	return newFromFile(path.Join(dir, configFile))
-+	return newFromFile(path.Join(dir, "shell-config.yml"))
++	return newFromFile("/run/gitlab/shell-config.yml")
```

Comment:
why was that necessary? To increase readability?

Reply:
reply from @globin via another channel: there were multiple callers of NewFromDir, und by patching it here we catch all.